### DISCUSSION TITLE
microsoft teams

### DISCUSSION THREAD

**Discussion by rbarbrow** [microsoft/winget-cli#1207] [Q&A] [answered]
Microsoft teams will not upgrade 100% of the time  "error: Installer failed with exit code: 1638"

is there a plan for Microsoft to get their product working 

OR

is there a way in settings to say skip upgrade on teams (or a different program)?

**Comment by denelon** [comment, marked as answer]
From the perspective of the Windows Package Manager, we have a few different work items in progress to address several upgrade scenarios that are currently failing. We're going to have a feature #476 to Pin a package. We're also looking at #1163 Enhancements to upgrade (Apps that upgrade themselves). We also have #1073 Schema extension to support multiple ARP entries.
